src/chw.js

```diff
@@ -551,7 +548,8 @@ go.app = function() {
                                     self.user.extra.ussd_sessions = '0';
                                     return Q.all([
                                         self.im.contacts.save(self.user),
-                                        self.im.contacts.save(self.contact)
+                                        self.im.contacts.save(self.contact),
+                                        go.utils.adjust_percentage_registrations(self.im, self.metric_prefix)
```

Comment:
I think ultimately we need to stop making server side calculated metrics and move to display time calcs for this kind of thing. Fine with this for now.